series.tex

```diff
@@ -0,0 +1,79 @@
+% Series expansion (Differentiate between the two approaches being used)
+\subsubsection{Series Expansion}
+
+SymPy is able to calculate the symbolic series expansion of an arbitrary series
+or expression involving elementary and special functions and multiple
+variables. For this it has two different implementations.
+
+The first approach stores a series as a core object. Each function has its
+specific implementation of its expansion which is able to evaluate the Puiseux
+series expansion about any point.
+
+\begin{verbatim}
+>>> from sympy import symbols, series
+>>> x, y = symbols('x, y')
+>>> series(sin(x+y) + cos(x*y), x, 0, 2)
+1 + sin(y) + x*cos(y) + O(x**2)
+\end{verbatim}
+
+The newer and much faster approach called Ring Series makes use of the
```

Comment:
There are a few speed comparisons [here](https://github.com/sympy/sympy/blob/master/doc/src/modules/polys/ringseries.rst). What sort of citation would do? 

Every series that ring series expands, it does so a couple of times faster than the default series method. We were planning to benchmark the two methods. What would be a good way to go about that?